modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItem.js

```diff
@@ -203,87 +300,133 @@ function EditItem() {
 
 				<div className="flex-grow-1 flex-shrink-1 overflow-auto p-4">
 					{options?.items?.length > 0 ? (
-						<ClayForm>
-							<Options
-								cartItemId={editedItem.cartItemId}
-								channelId={channel.id}
-								namespace={MINI_CART_NAMESPACE}
-								productId={editedItem.productId}
-								productOptions={options.items}
-								selectedItem={selectedItem}
-							/>
-						</ClayForm>
+						<>
+							<div className="panel panel-unstyled">
```

Comment:
Is the panel title always shown? (also if we don't display uoms?)

Reply:
Yes, it was already approved in the PM review